src/PageTransition.jsx

```diff
@@ -48,13 +48,19 @@ export default class PageTransition extends React.Component {
         const dom = ReactDom.findDOMNode(child);
         child.onTransitionDidEnd && child.onTransitionDidEnd(this.props.data);
         dom.classList.remove('transition-item');
+        this.actionClass && dom.classList.remove(this.actionClass);
       }
     } else {
       this.transite(this.props.children);
     }
   }
 
   componentWillReceiveProps(nextProps) {
+    if(nextProps.transitionAction && nextProps.transitionAction != ""){
+      this.actionClass = "transition-" + nextProps.transitionAction;
```

Comment:
Why do we need to set `this.actionClass`? Can we use the `this.props.transitionAction` directly in `transite` method? (like in https://github.com/trungdq88/react-router-page-transition/pull/39)

Reply:
Can do, was just trying to reduce repeated code. I'll switch it to use props inline.